src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -476,7 +476,7 @@ class StockMovement implements Validateable{
     }
 
     // Because this function needs to handle more than one enum, I left the argument's type as def
-    static String getTranslatedDisplayStatus(def status) {
+    String getTranslatedDisplayStatus(def status) {
```

Comment:
I didn't get a chance to review this but could you please change the implementation of this method to be ...
```
   return applicationTagLib.message(code: 'enum.' + status?.class?.simpleName + '.' + status)
```

Also why do we need a second method to get the translated display status when the original getDisplayStatus() method already does the localization?

Reply:
> Also why do we need a second method to get the translated display status when the original getDisplayStatus() method already does the localization?

I am not sure if I understand what you mean, but I moved this to a separate function because I wanted to make the code more clean :thinking: